amical/mf_pipeline/bispect.py

```diff
@@ -1016,6 +1019,20 @@ def _add_infos_header(infos, hdr, mf, pa, filename, maskname, npix):
     return infos
 
 
+def produce_result_pdf(figdir, filename):
+    # Call the PdfFileMerger
+    mergedObject = PdfFileMerger()
+
+    for fileNumber in range(7):
+        ifile = figdir + filename + "_" + str(fileNumber + 1) + ".pdf"
```

Comment:
This isn't really robust
```suggestion
        ifile = os.path.join(figdir, f"{filename}_{fileNumber + 1}.pdf")
```